pasqal_cloud/__init__.py

```diff
@@ -254,7 +256,8 @@ def rebatch(
             mas_runs: retry jobs with less or as many as this amount of runs.
             start_date: retry jobs created at or after this datetime.
             end_date: retry jobs created at or before this datetime.
-
+        Returns:
+            An instance of a Batch model
```

Comment:
What about "The instance of the re-created batch model" instead to make it more precise ?

Reply:
I'll add some more specificity